src/ToolbarServiceProvider.php

```diff
@@ -89,6 +94,27 @@ private function registerPublishing()
         }
     }
 
+    /**
+     * Determine if the application is handling an approved request.
+     *
+     * @param  \Illuminate\Foundation\Application  $app
+     * @return bool
+     */
+    private function runningApprovedRequest()
+    {
+        return ! $this->app->runningInConsole() && ! $this->app['request']->is(
+            array_merge([
+                config('telescope.path').'*',
+                'telescope-api*',
+                'vendor/telescope*',
+                'horizon*',
+                'vendor/horizon*',
+            ],
+            config('telescope.ignore_paths', []),
+            config('telescope-toolbar.ignore_paths'))
```

Comment:
Should this also default to an array (if it's not set?)

Reply:
I missed that one, fixed!